src/common/interfaces/campaign.ts

```diff
@@ -9,6 +9,7 @@ interface ICampaign {
 	image: string;
 	title: string;
 	deadline: Date;
+	deletedDate: Date;
```

Comment:
add a boolean to this model, isDeleted: true, alongside the deletion date to enable queries easily ignore deleted campaigns